File: 1-iniciante/1013/1013.cs
using System;

class URI {
    static void Main(string[] args) {
        int a, b, c, big;
        string[] str = Console.ReadLine().Split();
        a = int.Parse(str[0]);
        b = int.Parse(str[1]);
        c = int.Parse(str[2]);
        big = (a + b + Math.Abs(a - b)) / 2;
        big = (big + c + Math.Abs(big - c)) / 2;
        Console.WriteLine($"{big} eh o maior");
    }
}

File: 1-iniciante/1013/1013.c
#include <stdio.h>
#include <stdlib.h>

int main() {
    int a, b, c, big;
    scanf("%d%d%d", &a, &b, &c);
    big = (a + b + abs(a - b)) / 2;
    big = (big + c + abs(big - c)) / 2;
    printf("%d eh o maior\n", big);
    return 0;
}